### ISSUE TITLE
Example app crashes on IOS if the tag includes custom URL Scheme

### ISSUE THREAD

**Issue by LeeviVesa** [semlette/nfc_in_flutter#19]
When the app reads the NFC tag, the Core NFC reader displays success, but the application crashes. Probably something to do with IOS 13 background tag reading.

Tested with iPhone XR, ios 13,2

**Comment by semlette** [comment]
Hi!

I have pushed a commit to the `custom-url-fix` branch. Could you try using that and see if the issue persists?
You can set the nfc_in_flutter version in your pubspec.yaml to a Git branch like so:
```yaml
dependencies:
  nfc_in_flutter:
    git:
      url: git://github.com/semlette/nfc_in_flutter.git
      ref: custom-url-fix
```

**Comment by semlette** [comment]
The `custom-url-fix` branch fixed the issue when I tried to reproduce it, so I assume it will fix it for you as well. 
I have released version `2.0.2` on pub.dev with the fix.
